### ISSUE TITLE
Work only with local copy of package (Meteor 1.2.1)

### ISSUE THREAD

**Issue by comerc** [meteor-space/template-controller#13]
...else [reproduction](https://github.com/comerc/test-template-controller)

```
 ~/test-template-controller/.meteor/local/build/programs/server/packages/space_template-controller.js:29
   for (let key of Object.keys(handlers)) {                                    
            ^^^
 SyntaxError: Unexpected identifier
     at ~/test-template-controller/.meteor/local/build/programs/server/boot.js:241:30
     at Array.forEach (native)
     at Function._.each._.forEach (~/.meteor/packages/meteor-tool/.1.1.10.1q3nvqb++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
     at ~/test-template-controller/.meteor/local/build/programs/server/boot.js:137:5
=> Exited with code: 8
=> Your application is crashing. Waiting for file change.
```

**Comment by rhyslbw** [comment]
@comerc Have you tried using the latest _develop_ branch? #7 may have fixed this already

Best way to test this out is using [mgp](https://github.com/DispatchMe/mgp)

**Comment by DominikGuzei** [comment]
@comerc thanks for reporting, i will test this today but it is really strange because the package itself is version 1.2.1 and not 1.3 … and i would think that this error should not appear when using the `ecmascript` package.

**Comment by DominikGuzei** [comment]
@comerc i tested it with your repository and it seems to be fixed with `space:template-controller@0.2.2`
